packages/palette/src/helpers/injectGlobalStyles.tsx

```diff
@@ -129,6 +129,14 @@ export function injectGlobalStyles<P>(
       }
     }
 
+    h1, h2, h3, h4, h5, h6 {
+      font-style: inherit;
+      font-family: inherit;
+      font-weight: inherit;
+      font-size: inherit;
+      margin: 0;
+    }
+    
```

Comment:
@pepopowitz - would you mind taking care of this [bit of tech-debt](https://github.com/artsy/reaction/blob/master/src/Artsy/Router/Components/Boot.tsx#L36-L42)?

Reply:
I meant to mention that I'd clean that up, but forgot 😁

Reply:
ty 👍 cc @eessex